atomictests.py

```diff
@@ -0,0 +1,133 @@
+        )
+        self.assertTrue(
+            re.search(r'atomicos-docker--meta.*\d+(?:.\d+)?M.*lvm', out)
+        )
+        self.assertTrue(
+            re.search(r'atomicos-docker--data.*\d+(?:.\d+)?G.*lvm', out)
+        )
+
+
+@unittest.skipUnless(if_atomic(), "It's not an Atomic image")
+class TestDockerInstalled(unittest.TestCase):
+
+    def test_success(self):
+        out, err, eid = system('rpm -q docker')
+        out = out.decode('utf-8')
+        self.assertTrue('docker' in out)
+
+
+@unittest.skipUnless(if_atomic(), "It's not an Atomic image")
+class TestAtomicUpgradeRun(unittest.TestCase):
+
+    def test_upgrade_run(self):
+        out, err, eid = system('sudo rpm-ostree status')
+        out = out.decode('utf-8')
+        self.assertTrue(out)
+        out, err, eid = system('sudo ostree admin status')
+        out = out.decode('utf-8')
+        self.assertTrue(out)
+        out, err, eid = system(
+            'sudo cat /ostree/repo/refs/heads/ostree/0/1/0 > /etc/file1')
+        err = err.decode('utf-8')
+        self.assertFalse(err)
+        out, err, eid = system('sudo rpm-ostree upgrade')
+        err = err.decode('utf-8')
+        self.assertFalse(err)
+
+
+@unittest.skipUnless(if_atomic(), "It's not an Atomic image")
+class TestAtomicUpgradePostReboot(unittest.TestCase):
+
+    def test_upgrade_post_reboot(self):
+        out, err, eid = system(
+            'docker run --rm busybox true && echo "PASS"')
+        out = out.decode('utf-8')
+        self.assertEquals('PASS\n', out)
+
+
+@unittest.skipUnless(if_atomic(), "It's not an Atomic image")
+class TestAtomicRollbackRun(unittest.TestCase):
+
+    def test_atomic_rollback_run(self):
+        out, err, eid = system(
+            'sudo cat /ostree/repo/refs/heads/ostree/1/1/0 > /etc/file2')
+        err = err.decode('utf-8')
+        self.assertFalse(err)
+
+        out, err, eid = system('sudo rpm-ostree rollback')
+        err = err.decode('utf-8')
+        self.assertFalse(err)
+
+
+@unittest.skipUnless(if_atomic(), "It's not an Atomic image")
+class TestAtomicRollbackPostReboot(unittest.TestCase):
+
+    def test_atomic_rollback_post_reboot(self):
+        out, err, eid = system('rpm-ostree status')
+        out = out.decode('utf-8')
+        self.assertTrue(out)
+
+        out, err, eid = system('sudo cat /etc/file1')
+        out = out.decode('utf-8')
+        self.assertTrue(out)
+
+        out, err, eid = system('sudo cat /etc/file2')
+        err = err.decode('utf-8')
+        self.assertTrue(err)
+
+        out, err, eid = system(
+            'docker run --rm busybox true && echo "PASS"')
+        out = out.decode('utf-8')
+        self.assertEqual(out, 'PASS\n')
+
+
+@unittest.skipUnless(if_atomic(), "It's not an Atomic image")
+class TestAtomicDockerImage(unittest.TestCase):
+
+    def test_atomic_image(self):
+        out, err, eid = system(
+            'curl -O https://dl.fedoraproject.org/pub/alt/stage/23_TC1/Docker'
```

Comment:
Any place we can put this URL so that we can easily change it in the future? Some constants file maybe?